src/Reflection/ReflectionConstant.php

```diff
@@ -140,10 +140,7 @@ public function getName(): string
             return $this->getNameFromDefineFunctionCall($this->node);
         }
 
-        /**
-         * @psalm-suppress UndefinedPropertyFetch
```

Comment:
Can we add `findUnusedPsalmSuppress="true"` to `psalm.xml`?

Reply:
Done.

Reply:
Awesome, thanks!